libs9s/s9srpcclient.cpp

```diff
@@ -7204,7 +7208,19 @@ S9sRpcClient::upgradeCluster()
     jobData["clusterid"]  = clusterId;
     if (hosts.size() != 0)
     {
-	jobData["nodes"]      = nodesField(hosts);
+	    jobData["nodes"]      = nodesField(hosts);
+    }
+    if (upgradeToVersion != "")
+    {
+        jobData["upgrade_to_version"]    = upgradeToVersion;
```

Comment:
maybe better in the lines like this one to place a single whitespace before the equal mark: ` = `
each `if` in this code snippet has its own scope, no need to align values across all such `if` scopes